### PR TITLE
Retrieve max ten users from API call

### PR DESCRIPTION
**Pull request by cbelan-d2l**
* Add a couple more routes that will be used soon

### CODE REVIEW

File: src/helpers/leaderboardRoutes.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
-export const BasVersion = 1.1;
+const BasVersion = 1.1;
```
Comment (ryoung-d2l) [inline src/helpers/leaderboardRoutes.js]: Bas?

File: src/components/awards-leaderboard-ui.js [hunk 2]
```diff
@@ -63,7 +63,7 @@ class App extends BaseMixin(LitElement) {
 	async _getLeaderboard() {
 		const myLeaderboard = await LeaderboardService.getLeaderboard(this.orgUnitId);
 		console.log(myLeaderboard); // eslint-disable-line no-console
```
Comment (ryoung-d2l) [inline src/components/awards-leaderboard-ui.js]: don't forget about this console.log later

File: src/helpers/leaderboardRoutes.js [hunk 3]
```diff
@@ -1,6 +1,9 @@
-export const BasVersion = 1.1;
+const BasVersion = 1.1;
+const MAX_USERS = 10;
 
 export class LeaderboardRoutes {
-	static ClasslistLeaderboard(orgUnitId) { return `/d2l/api/bas/${BasVersion}/orgunits/${orgUnitId}/classlist/`; }
+	static ClasslistLeaderboard(orgUnitId) { return `/d2l/api/bas/${BasVersion}/orgunits/${orgUnitId}/classlist/?limit=${MAX_USERS}`; }
```
Comment (depowell-D2L) [inline src/helpers/leaderboardRoutes.js]: Not strictly needed but you could use arrow notation to make it simpler:

```
static ClasslistLeaderboard = (orgUnitId) => `/d2l/api/bas/${BasVersion}/orgunits/${orgUnitId}/classlist/?limit=${MAX_USERS}`;
```

File: src/helpers/leaderboardRoutes.js [hunk 4]
```diff
@@ -1,6 +1,9 @@
-export const BasVersion = 1.1;
+const BasVersion = 1.1;
+const MAX_USERS = 10;
```
Comment (depowell-D2L) [inline src/helpers/leaderboardRoutes.js]: Why have 1 const as all caps and the other as title case? (`MAX_USER` vs `BasVersion`)

File: src/helpers/leaderboardRoutes.js (hunk 1 above)
Comment (cbelan-d2l) [reply src/helpers/leaderboardRoutes.js]: Brightspace Awards Service
If you look at the API calls, they all begin with it
`/d2l/api/bas/${BasVersion}/`...

File: src/components/awards-leaderboard-ui.js (hunk 2 above)
Comment (cbelan-d2l) [reply src/components/awards-leaderboard-ui.js]: Yes - This will be removed in future PR

File: src/helpers/leaderboardRoutes.js (hunk 4 above)
Comment (cbelan-d2l) [reply src/helpers/leaderboardRoutes.js]: Oops!

**Pull request merged**